src/Bicep.Core.UnitTests/Configuration/ConfigurationManagerTests.cs

```diff
@@ -106,6 +106,7 @@ public void GetBuiltInConfiguration_NoParameter_ReturnsBuiltInConfigurationWithA
           }
         },
         "experimentalFeaturesEnabled": {
+          "extendableParamFiles": false,
```

Comment:
Please could you make sure we have an explicit test to ensure this feature cannot be used unless the experimental feature is enabled?

Reply:
I added a test in ConfigurationManagerTests, and another one in BuildParamsCommandTests, to test the scenario you mentioned 👍